components/Sidebar/SidebarContent.js

```diff
@@ -70,9 +68,19 @@ const SidebarContent = ({ onClose, mode, ...rest }) => {
           onClick={onClose}
         />
       </Flex>
+      <Flex h="14" alignItems="center" mx="8" justifyContent="space-between">
+        <Text textColor="white" fontSize="2xl" fontWeight="normal">
+          {mode}
+        </Text>
+      </Flex>
       <Box mt={50}>
         {links.map((link) => (
-          <NavItem key={link.name} icon={link.icon} href={link.href}>
+          <NavItem 
+                  key={link.name} 
+                  icon={link.icon} 
+                  href={link.href}
+                  bg={mode === "mentor"?"brand.blue":"brand.green"}
```

Comment:
Looks like we reuse this logic for the background setup. Maybe it makes sense to extract it in a reusable variable

Reply:
totally right, it will be part of a clean up later on